core/drivers/crypto/caam/sub.mk

```diff
@@ -9,4 +9,4 @@ srcs-y += caam_jr.c
 srcs-y += caam_rng.c
 srcs-y += caam_desc.c
 subdirs-$(CFG_NXP_CAAM_HASH_DRV) += hash
-
+subdirs-$(CFG_NXP_CAAM_MKVB_DRV) += mkvb
```

Comment:
master key derivation is a particular CAAM BLOB operation. It would be better for future blob operation to rename the mkvb to blob (folder like CFG define)

Reply:
Will do.

Reply:
@Emantor Do you agree on renaming it to ```blob``` instead of ```mkvb``` ?